azkaban-common/src/main/java/azkaban/project/AzkabanProjectLoader.java

```diff
@@ -184,35 +184,39 @@ private boolean isReportStatusValid(final Map<String, ValidationReport> reports,
   private void persistProject(final Project project, final FlowLoader loader, final File archive,
       final User uploader) throws ProjectManagerException {
     synchronized (project) {
+      final int newVersion = this.projectLoader.getLatestProjectVersion(project) + 1;
+      final Map<String, Flow> flows = loader.getFlowMap();
+      for (final Flow flow : flows.values()) {
+        flow.setProjectId(project.getId());
+        flow.setVersion(newVersion);
+      }
+
+      this.storageManager.uploadProject(project, newVersion, archive, uploader);
+
+      log.info("Uploading flow to db " + archive.getName());
```

Comment:
"Uploading flow"  + archive.getName + " to db"

Reply:
are we uploading flow or project here?

Reply:
Uploading flows to DB project_flows table. 
archive.getName is the project zip name. The logging message is not very clear.